aries-site/src/examples/components/checkbox/CheckBoxDisabledExample.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Box, CheckBox, Form, FormField } from 'grommet';
+
+export const CheckBoxDisabledExample = () => {
+  return (
+    <Box gap="large" direction="row">
```

Comment:
Mobile layout needs to be fixed. row-responsive?
![Screen Shot 2020-04-28 at 12 22 22 PM](https://user-images.githubusercontent.com/1756948/80523314-18cf1b00-894b-11ea-8210-f6515f560754.png)